docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -50,6 +50,9 @@ metastore service (HMS) or AWS Glue. The catalog type is determined by the
 ``iceberg.catalog.type`` property, it can be set to either ``HIVE_METASTORE``
 or ``GLUE``.
 
+
+.. _iceberg-hive-catalog:
+
 Hive metastore catalog
```

Comment:
Change to 

Hive metastore service (HMS) 

or just 

Hive metastore

Reply:
This is about the Iceberg catalog implementation which is backed by HMS. In this case I tend to think that the name is suitable. NO

Reply:
Sure .. but in the context of Trino its confusing to use catalog for two different concepts .. not sure how to get around this better